### PR TITLE
Accessibility automation

### PR DESCRIPTION
**Pull request by VaclovasV**

### CODE REVIEW

File: gulp-tasks/image-optimization.js [hunk 1]
```diff
@@ -1,12 +1,49 @@
 module.exports = function () {
-	var imagemin = require('gulp-imagemin'),
-		cache = require('gulp-cache'),
-		gulp = require('gulp'),
-		config = require('../gulp.config.js')();
+    var imagemin = require('gulp-imagemin'),
```
Comment (kestas-devbridge) [inline gulp-tasks/image-optimization.js]: Should be `const`

File: gulp-tasks/image-optimization.js [hunk 2]
```diff
@@ -1,12 +1,49 @@
...
+            }),
+            //gif
+            // imagemin.gifsicle({
+            //     interlaced: true,
+            //     optimizationLevel: 3
+            // }),
+            //gif very light lossy, use only one of gifsicle or Giflossy
+            imageminGiflossy({
+                optimizationLevel: 3,
+                optimize: 3, //keep-empty: Preserve empty transparent frames
```
Comment (kestas-devbridge) [inline gulp-tasks/image-optimization.js]: Should be string: `'3'`

File: test-accessibility.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
+// Automated accessibility testing script
+'use strict';
+
+const pa11y = require('pa11y');
+const htmlReporter = require('pa11y-reporter-html');
+const puppeteer = require('puppeteer');
+var fs = require('fs');
```
Comment (kestas-devbridge) [inline test-accessibility.js]: Should be `const`

File: test-accessibility.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+const reportsPath = 'reports/';
```
Comment (kestas-devbridge) [inline test-accessibility.js]: Maybe move to config?

File: test-accessibility.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    let report = [];
+    const hostPort = process.argv[2] || 4200;
+    const hostName = 'devbridge.com';
+    const hostUrl = 'https://' + hostName;
+    let hasErrors = false;
+
+    try {
+        browser = await puppeteer.launch();
+
+        const testableUrls = [
```
Comment (kestas-devbridge) [inline test-accessibility.js]: Maybe move to config or add info at README?

File: test-accessibility.js [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+                method: 'GET',
+                ignore: [
+                    // Ignored because test cannot detect button type if it's not rendered in DOM.
+                    'WCAG2A.Principle3.Guideline3_2.3_2_2.H32.2'
+                ],
+                headers: {
+                    accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8',
+                    host: hostName
+                },
+                //screenCapture: `./accessibility/screen${index}.png`,
```
Comment (kestas-devbridge) [inline test-accessibility.js]: Is path correct?

File: test-accessibility.js (hunk 3 above)
Comment (VaclovasV) [reply test-accessibility.js]: fixed

File: test-accessibility.js (hunk 4 above)
Comment (VaclovasV) [reply test-accessibility.js]: it is gulp config, so not very related, good idea would be to have constants, but I think, this is way tof rimprovement :)

File: test-accessibility.js (hunk 6 above)
Comment (VaclovasV) [reply test-accessibility.js]: removed functionality which was brought from my old proejct

File: test-accessibility.js (hunk 5 above)
Comment (VaclovasV) [reply test-accessibility.js]: it is gulp config, so not very related, good idea would be to have constants, but I think, this is way for improvement :)

Comment (kestas-devbridge) [reply test-accessibility.js]: Agree with config but this section should be empty array as now we do not have these pages. Also maybe add command to README with note that user has to define urls here?

**Pull request merged**